### PR TITLE
Updated selectProduct page UI(#29pux36)

### PR DESCRIPTION
**Pull request by disha1202**
Screenshot from 2022-05-04 19-14-45
**Updated screenshot:**
image

### CODE REVIEW

File: src/views/SelectProduct.vue [hunk 1]
```diff
@@ -17,73 +17,84 @@
...
         <aside class="filters desktop-only">
           <ion-list>
-            <ion-list-header><h3>{{ $t("Include") }}</h3></ion-list-header>
+            <ion-item lines="inset">
+              <ion-label position="fixed">{{ $t("Threshold") }}</ion-label>
+              <ion-input type="text" :placeholder="$t('global threshold')"/>
+            </ion-item>
+            <ion-list-header>
+              <ion-label>{{ $t("Include") }}</ion-label>
+              <ion-button fill="clear" color="warning">RESET</ion-button>
```
Comment (adityasharma7) [inline src/views/SelectProduct.vue:29]: All label and static text should be internationalised. Please update other such instances as well

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/SelectProduct.vue [hunk 2]
```diff
@@ -17,73 +17,84 @@
...
-        </section>
 
         <aside class="filters desktop-only">
           <ion-list>
-            <ion-list-header><h3>{{ $t("Include") }}</h3></ion-list-header>
+            <ion-item lines="inset">
+              <ion-label position="fixed">{{ $t("Threshold") }}</ion-label>
+              <ion-input type="text" :placeholder="$t('global threshold')"/>
+            </ion-item>
+            <ion-list-header>
```
Comment (adityasharma7) [inline src/views/SelectProduct.vue:27]: Revert the markup

File: src/views/SelectProduct.vue [hunk 3]
```diff
@@ -17,73 +17,84 @@
...
+              <ion-label>{{ $t("Include") }}</ion-label>
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:28]: H3 label was replaced with ion label incorrectly.

File: src/views/SelectProduct.vue [hunk 4]
```diff
@@ -17,73 +17,84 @@
...
       <div class="find">
-        <section class="search">
-          <ion-searchbar :placeholder="$t('Search products')" />
-        </section>
 
         <aside class="filters desktop-only">
           <ion-list>
-            <ion-list-header><h3>{{ $t("Include") }}</h3></ion-list-header>
+            <ion-item lines="inset">
+              <ion-label position="fixed">{{ $t("Threshold") }}</ion-label>
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:24]: do not need to define position. default value is fine.

File: src/views/SelectProduct.vue (hunk 1 above)
Comment (dt2patel) [reply src/views/SelectProduct.vue:29]: label also does not need to be in all caps. styling takes care of that.

File: src/views/SelectProduct.vue [hunk 5]
```diff
@@ -17,73 +17,84 @@
...
+            </ion-item>
+            <ion-list-header>
+              <ion-label>{{ $t("Include") }}</ion-label>
+              <ion-button fill="clear" color="warning">RESET</ion-button>
+            </ion-list-header>
             <ion-card>
               <ion-toolbar>
                 <ion-title>{{ $t("Tags") }}</ion-title>
+                <ion-button fill="clear" slot="end">
+                 <ion-label>Add</ion-label>
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:35]: @disha1202 if you made previous button label all caps, then this should also be all caps. Code patterns, even when wrong, should be consistent. Inconsistent code just as bad as broken code.

File: src/views/SelectProduct.vue [hunk 6]
```diff
@@ -17,73 +17,84 @@
...
+              <ion-input type="text" :placeholder="$t('global threshold')"/>
+            </ion-item>
+            <ion-list-header>
+              <ion-label>{{ $t("Include") }}</ion-label>
+              <ion-button fill="clear" color="warning">RESET</ion-button>
+            </ion-list-header>
             <ion-card>
               <ion-toolbar>
                 <ion-title>{{ $t("Tags") }}</ion-title>
+                <ion-button fill="clear" slot="end">
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:34]: button size missing. Need to use a small button.

File: src/views/SelectProduct.vue [hunk 7]
```diff
@@ -17,73 +17,84 @@
...
+            <ion-item lines="inset">
+              <ion-label position="fixed">{{ $t("Threshold") }}</ion-label>
+              <ion-input type="text" :placeholder="$t('global threshold')"/>
+            </ion-item>
+            <ion-list-header>
+              <ion-label>{{ $t("Include") }}</ion-label>
+              <ion-button fill="clear" color="warning">RESET</ion-button>
+            </ion-list-header>
             <ion-card>
               <ion-toolbar>
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:32]: Use ion-item with ion label and button in slot end. Available in figma for reference as well

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/SelectProduct.vue (hunk 7 above)
Comment (meet-aniket) [reply src/views/SelectProduct.vue:32]: Updated code Sir.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**